src/wallets/ethereum.ts

```diff
@@ -108,6 +108,14 @@ export class Web3EthereumWallet implements EthereumWallet {
     return this.signAndSend(tx);
   }
 
+  public async getLatestBlockTimestamp() {
+    logger.info("Retrieving latest block");
+    const latest = await this.web3.eth.getBlockNumber();
+    const block = await this.web3.eth.getBlock(latest);
+
+    return +block.timestamp;
```

Comment:
why the `+`?

Reply:
So that a string is converted to a number.